sell/sell-api.yaml

```
@@ -167,7 +167,17 @@ paths:
             type: string
         - name: type
           in: query
-          description: The type of activity. One of `NOTE`, `MEETING`, `CALL`, `TASK`, `EMAIL`, or `EVENT`.
+          description: The type of activity. USER or EVENT.
+          schema:
+            type: string
+        - name: activityType
+          in: query
+          description: The type of user activity. One of `NOTE`, `MEETING`, `CALL`, `TASK`, etc.
+          schema:
+            type: string
+        - name: eventType
+          in: query
+          description: The type of event activity. 'COMPANY_CREATED', 'QUOTE_CREATED', etc.
           schema:
             type: string
         - name: associatedContactId
```

```suggestion
        - name: associatedContactIds
```
To match with the current Activity entity. Same for the other fields

---

Probably here should be singular because we are looking for the one associate contacted, no?